src/mock/handlers/invite.ts

```diff
@@ -0,0 +1,23 @@
+import { rest } from 'msw';
+
+export default [
+    rest.post('/api/invite', (req, res, ctx) => {
+        return res(ctx.status(200));
+    }),
+    rest.get('/api/get-invite-data', (req, res, ctx) => {
+        const responseBody = {
+            MoC: 'Test MoC',
+            topic: 'Test Topic',
+            eventDateTime: new Date().toUTCString(),
+            constituentScope: 'state',
+            region: 'Test Region',
+            townHallId: 'Test TownHallID',
+        };
+        return res(ctx.json(responseBody), ctx.status(200));
+    }),
```

Comment:
Don't forget to make sure your components handle error responses appropriately.  As you know, things don't always succeed.

Reply:
I handled this as a special case in the story since none of the data passed itself should influence the response. I force the worker to give a 400 response if the status control is set to fail.